test/library/options-test.ts

```diff
@@ -50,6 +37,30 @@ describe('options test', () => {
 		expect(store.options.draft_polli_ratelimit_headers).toEqual(true)
 	})
 
+	// TODO: Update in v8.
```

Comment:
I think we're probably going to keep `max` around forever. It's been both correct and more-or-less required for essentially every release up to now, so I think removing it or even adding warnings is probably too much.

Reply:
Makes sense, removed the comment in 2a0d80d.